### ISSUE TITLE
Doorperson - Scanner - Successful Scan Vibration

### ISSUE THREAD

**Issue by Mcscott142** [revelrylabs/bn-mobile-react#611]
## Background
We should have the phone vibrate on a successful scan. I thought we were already doing this. Maybe not or maybe it got reverted somehow. 

## Scenario: A doorperson scans a valid ticket
Given a ticket is valid
- [ ] When I scan a ticket
- [ ] Then my device vibrates once to let me know the scan was successful

**Comment by jwietelmann** [comment]
Can you please triple-check that? The code is all still here.

```javascript
  // trigger the final check-in success or failure feedback, wait a little bit, then reset
  _finishCheckIn(error = null) {
    let delay

    if (error) {
      this.setState({error})
      vibe.sad()
      delay = DELAY_SLOW
    } else {
      vibe.happy()
      delay = this.state.checkInMode === 'automatic' ? DELAY_SLOW : DELAY_FAST
    }
```

**Comment by Mcscott142** [comment]
I can, assigned myself and moved to Icebox. Colin to test this on Android and iOS

**Comment by Mcscott142** [comment]
@zmelamed I tested on Android and iOS and I'm getting the successful vibration scan. Can you provide some more details around the issue they were having? 

Maybe we need more than a single vibration? 
Was it everyone?
What type of device?
Did they check their Vibration settings on the device?

**Comment by Mcscott142** [comment]
I think this was resolved with #617 and adding permissions for vibrations to android. It's vibrating for me when testing on the latest stage build. Closing this one out.
